overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
+  file, err := os.Open(path)
+  defer file.Close()
+  if err != nil {
+    fmt.Printf("error opening file: %v\n",err)
+    return overrides
+  }
+
+  reader := bufio.NewReader(file)
+
+  var line string
+  for {
+    line, err = reader.ReadString('\n')
+    line_contents := strings.TrimSpace(line)
+    if strings.HasPrefix(line_contents, "#"){
+      // The line is a comment, so we'll skip it
+      continue
+    }
+    if line_contents == ""{
+      break
+    }
+
+    elements := strings.Split(line_contents, ",")
```

Comment:
May want to check the number of elements we get back from Split here to avoid potentially indexing to a nonexistent index in elements

Reply:
Added in https://github.com/ameihm0912/geomodel/pull/10/commits/7924535a214c3750e5ab7e2b4d0e4058e63dc2c1